spec/exercises.js

```diff
@@ -44,18 +50,23 @@ describe('Testing exercises', function () {
   var exercises = exec.sync(['list', '--lang=en']).toString().split('\n')
   exercises.pop() // last line break
   exercises.map(function (exercise) {
-    return wUtil.idFromName(exercise)
-  }).forEach(function (id, nr) {
-    var folder = path.join(process.cwd(), 'test', id)
-    var allFiles = require('glob').sync('*.*', {
+    return {
+      name: exercise,
+      id: wUtil.idFromName(exercise, config.spaceChar)
+    }
+  }).forEach(function ({ name, id }, nr) {
```

Comment:
Destructuring is a Node 6 feature. if we start using this we are setting precedent to use all Node 6 + language features, like arrow functions. Not sure what to do with this.

Reply:
I'm agree to update the code to ES6. I think we can start changing small parts like variables, functions, etc .. that don't require transpile code.

In order to get an easy review I think changes don't have to involve more than an exercise for PR.